index.html

```diff
@@ -0,0 +1,181 @@
+                        <li>Javascript</li>
+                        <li>html</li>
+                    </ul>
+                 <input type="submit" value="See Project" class="btn"/>
+            </div>
+        </div>
+        <div class="card">
+            <div class="works-heading"></div>
+            <div class="works-footer">
+                <p class="Project-title">Multi-Post stories Gain+Glory</p>
+                    <ul class="list-inline">
+                        <li>Ruby on rails</li>
+                        <li>css</li>
+                        <li>Javascript</li>
+                        <li>html</li>
+                    </ul>
+                <input type="submit" value="See Project" class="btn"/>
+            </div>
+        </div>
+        <div class="card">
+            <div class="works-heading"></div>
+            <div class="works-footer">
+                <p class="Project-title">Multi-Post stories Gain+Glory</p>
+                    <ul class="list-inline">
+                        <li>Ruby on rails</li>
+                        <li>css</li>
+                        <li>Javascript</li>
+                        <li>html</li>
+                    </ul>
+                 <input type="submit" value="See Project" class="btn"/>
+            </div>
+        </div>
+
+        <div class="card">
+            <div class="works-heading"></div>
+            <div class="works-footer">
+                <p class="Project-title">Multi-Post stories Gain+Glory</p>
+                    <ul class="list-inline">
+                        <li>Ruby on rails</li>
+                        <li>css</li>
+                        <li>Javascript</li>
+                        <li>html</li>
+                    </ul>
+                 <input type="submit" value="See Project" class="btn"/>
+            </div>
+        </div>
+        <div class="card">
+            <div class="works-heading"></div>
+            <div class="works-footer">
+                <p class="Project-title">Multi-Post stories Gain+Glory</p>
+                    <ul class="list-inline">
+                        <li>Ruby on rails</li>
+                        <li>css</li>
+                        <li>Javascript</li>
+                        <li>html</li>
+                    </ul>
+                 <input type="submit" value="See Project" class="btn"/>
+            </div>
+        </div>
+        <div class="card">
+            <div class="works-heading"></div>
+            <div class="works-footer">
+                <p class="Project-title">Multi-Post stories Gain+Glory</p>
+                    <ul class="list-inline">
+                        <li>Ruby on rails</li>
+                        <li>css</li>
+                        <li>Javascript</li>
+                        <li>html</li>
+                    </ul>
+                 <input type="submit" value="See Project" class="btn"/>
+            </div>
+        </div>
+    </div>
+</div>
+<div class="about">
```

Comment:
- [ ] As mentioned by the previous reviewer the `About me` is in the `section` tag not in the `div` tag for better semantic HTML, kindly fix it :upside_down_face: :upside_down_face: